src/main/java/javax/jmdns/impl/JmDNSImpl.java

```diff
@@ -470,7 +470,10 @@ private void openMulticastSocket(HostInfo hostInfo) throws IOException {
             }
         }
         _socket.setTimeToLive(255);
-        _socket.joinGroup(_group);
```

Comment:
please update this fix in gradle..
the following gradle library is still a 2011 release
https://mvnrepository.com/artifact/javax.jmdns/jmdns/3.4.1